docs/prow/create-component-jobs.md

```diff
@@ -179,9 +176,9 @@ The trigger for ProwJobs are GitHub events. For example, GitHub sends events to
 To test your changes, create a new PR for your component.
 If you want to trigger your job again, add a comment on the PR for your component:
 
-- `/test all` to run all tests
+- `/test all` to rerun all tests
 - `/retest` to only rerun failed tests
-- `/test {your test name}`, such as `/test kyma-components-binding-usage-controller`, to only run a specific test
+- `/test {test-name}` or `/retest {test-name}` to only rerun a specific test. For example, run `/test kyma-components-binding-usage-controller`.
```

Comment:
This could also be `pre-master-kyma-components-binding-usage-controller` for consistency.